csrc/cpu/cpu_types_arm.hpp

```diff
@@ -0,0 +1,457 @@
+#include <arm_neon.h>
+#include <torch/all.h>
+
+#include <cmath>
+
+namespace vec_op {
+
+// FIXME: FP16 is not fully supported in Torch-CPU
```

Comment:
@sanketkaleoss do you have plans to support FP16 in a future PR? I see that it's partially implemented. 

Do you know what the problem with FP16 in Torch-CPU is?

Reply:
I'm not sure what's the problem there. I can add the support in future PR if torch-CPU supports FP16 in future.